tests/run-make/emit-notify/rmake.rs

```diff
@@ -0,0 +1,37 @@
+extern crate run_make_support;
+
+use run_make_support::{rustc, tmp_dir};
+
+fn main() {
+    let inc_dir = tmp_dir();
+
+    for _ in 0..=1 {
+        let output = rustc()
+            .input("lib.rs")
+            .arg("--emit=obj,asm,llvm-ir,llvm-bc,mir")
```

Comment:
Could you introduce some semantically meaningful helpers like

```
rust()
    .emit("obj,asm,llvm-ir,...")
    .codegen_units(1)
    .json("artifacts")
    .error_format("json")
    .incremental(inc_dir) // incremental<P: AsRef<Path>>(path: P)
```

Reply:
> Could you introduce some semantically meaningful helpers like

Sure. Here or in a separate pull request?

Reply:
> Sure. Here or in a separate pull request?

In this PR is fine, since you'll be using the helpers in this test